cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -248,6 +248,21 @@ def after_fill(self, was_change):
             wait_for(lambda: not self.save.disabled, timeout='10s', delay=0.2)
 
 
+class CopyCatalogItemView(ServicesCatalogView):
+    name = Input(name="name")
+
```

Comment:
**Suggestion:**
Remove this extra line.

Reply:
aaaah.... as per me when we write any `view` or you can say any `code`; to separate out difference generally we add such `blanks lines`. If you want i would like to remove :+1:

Reply:
Yes you're right approach but we can have blank line between group of `objects` and `property/method`, So I would suggest remove it :smile:

Reply:
hummm interesting :). @sbulage  can you please point me any guideline or python code styling instructions. I'm little bit confused about it. I found such thing in c-python code base as well :)